src/views/SignTransactionSuccess.vue

```diff
@@ -4,30 +4,40 @@
         <h1>Your transaction<br>is ready!</h1>
         <div style="flex-grow: 1;"></div>
         <button @click="done" :disabled="!isTxPrepared">Send now</button>
```

Comment:
This Button does not send the transaction but gives the SignTransactionResult to the caller. What he does with it is out of reach.

Reply:
You are right, it needs to be relabelled. But then again, this whole page has to be changed to adhere to the designs. But you are still right.